src/rev_ai/apiclient.py

```diff
@@ -109,6 +110,7 @@ def submit_job_url(
         :param notification_config: CustomerUrlData object containing the callback url to
             invoke on job completion as a webhook and optional authentication headers to use when
             calling the callback url
+        :param skip_postprocessing: skip all text postprocessing (punctuation, capitalization, ITN)
```

Comment:
What do you think about writing out ITN? Is this a commonly known acronym?

Reply:
I agree that ITN is confusing